### PR TITLE
Example working helm chart

### PR DESCRIPTION
**Pull request by idjohnson**
I included a basic helm values as well as my example helm values (values.yaml.example)

This assumes creating a namespace and a working Redis/Valkey and PostgreSQL instance

I used
```
kubectl create ns docmost
helm install redis bitnami/redis --namespace docmost --set auth.enabled=false
helm install postgresql bitnami/postgresql --namespace docmost
```

Note, i did attempt to create the database and set the user/password (but did not work)

```
$ helm install postgresql bitnami/postgresql --namespace docmost \
  --set postgresqlUsername=docmost \
  --set postgresqlPassword=somecomplicatedpassword\
  --set postgresqlDatabase=docmost
```

I just wanted to give back if you found it at all useful

### CODE REVIEW

**Comment by Justman100** [comment]
@sidnelui-krystal @wuast94 @Philipinho

**Comment by Philipinho** [comment]
Thank you for your contribution.
Ideally, I would like to keep more infrastructural code in a separate repo. I am sure many will find this useful. 
Closing for now.

**Comment by Justman100** [comment]
Nice that one simply deletes comments from others! I will probably not participate here...

**Comment by Philipinho** [comment]
Hi @Justman100, you commented on two separate PRs, mentioning unrelated users without providing context.  I could take one for a mistake, but two comments seem like spam.

**Comment by Justman100** [comment]
> Hi @Justman100, you commented on two separate PRs, mentioning unrelated users without providing context. I could take one for a mistake, but two comments seem like spam.

That was intentional! I marked it so that issues and PRs could be processed! And users who have write permissions here are unrelated?

**Comment by isaacjohnson-wellsky** [comment]
Tried various ways to do that CLA but it just errors for me
image

**Comment by isaacjohnson-wellsky** [comment]
All good - you can take the variable or not

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```
Comment (UnoYakshi) [inline helm/Chart.yaml:18]: I wonder what's a usual approach to bump the version here?

- Manually each time?
- With Semantic Release-like software?
- Or, perhaps, there is a built-in Helm solution?

Update: ah, this is the chart version. So, separate versioning… But my question is still relevant towards `appVersion` then.

**Pull request closed without merging**